src/util.js

```diff
@@ -12,5 +12,15 @@ module.exports = {
     return _.every(collection, function (item) {
       return _.isString(item);
     });
+  },
+
+  isArrayOfArrays: function (collection) {
+    return _.isArray(collection) && _.every(collection, function (item) {
+      return _.isArray(item);
+    });
+  },
```

Comment:
yeah, totally, but the prop this is needed for takes either an array or an array of arrays :/